src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/PaymentResult.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿
```

Comment:
What's this view for?  Isn't this the screen Steve will have been working on

Reply:
Removed.